vochain/scrutinizer/process.go

```diff
@@ -37,8 +37,18 @@ func (s *Scrutinizer) ProcessList(entityID []byte, fromID []byte, max int64) ([]
 		if !fromLock {
 			keyCopy := make([]byte, len(process))
 			copy(keyCopy, process)
-			processListResult = append(processListResult, keyCopy)
-			max--
+			if p, err := s.VochainState.Process(keyCopy, true); err != nil {
+				log.Debugf("error fetching process %x: %s", p.ProcessId, err)
+				continue
```

Comment:
If there is a continue here the else is not required....

Reply:
same as before